### PR TITLE
Support for Offset Noise in examples

### PR DESCRIPTION
**Pull request by haofanwang**
As title.

Offset noise is a recent training trick introduced by [crosslabs](https://www.crosslabs.org/blog/diffusion-with-offset-noise).

### CODE REVIEW

File: examples/text_to_image/train_text_to_image.py [hunk 1]
```diff
@@ -297,6 +297,7 @@ def parse_args():
     parser.add_argument(
         "--enable_xformers_memory_efficient_attention", action="store_true", help="Whether or not to use xformers."
     )
+    parser.add_argument("--noise_offset", type=float, default=0, help="The scale of noise offset.")
```
Comment (patrickvonplaten) [inline examples/text_to_image/train_text_to_image.py:300]:
```suggestion
    parser.add_argument("--noise_offset", type=float, default=0, help="The scale of noise offset. See: https://www.crosslabs.org//blog/diffusion-with-offset-noise for more information.")
```

File: examples/text_to_image/train_text_to_image_lora.py [hunk 2]
```diff
@@ -333,6 +333,7 @@ def parse_args():
     parser.add_argument(
         "--enable_xformers_memory_efficient_attention", action="store_true", help="Whether or not to use xformers."
     )
+    parser.add_argument("--noise_offset", type=float, default=0, help="The scale of noise offset.")
```
Comment (patrickvonplaten) [inline examples/text_to_image/train_text_to_image_lora.py:336]:
```suggestion
    parser.add_argument("--noise_offset", type=float, default=0, help="The scale of noise offset. See: https://www.crosslabs.org//blog/diffusion-with-offset-noise for more information.")
```

**Review by sayakpaul — APPROVED** [review]
Superb!

Just curious -- have you performed any experiments with noise offsets?

**Comment by sayakpaul** [comment]
Ah interesting!

I think once the code quality checks pass, we can merge this.

**Comment by haofanwang** [comment]
Done. @sayakpaul

**Comment by sayakpaul** [comment]
Excellent! Well done 🔥

**Pull request merged**